pyensembl/transcript.py

```diff
@@ -400,7 +400,10 @@ def sequence(self):
         Spliced cDNA sequence of transcript
         (includes 5" UTR, coding sequence, and 3" UTR)
         """
-        return self.genome.transcript_sequences.get(self.id)
+        for transcript_sequences in self.genome.transcript_sequences:
+            if self.id in transcript_sequences:
+                return transcript_sequences.get(self.id)
```

Comment:
Ah, I guess one issue with combining everything into transcripts.pickle is backwards compatibility. The change would require everyone to "re-pickle" their transcript and protein sequences.